client.go

```diff
@@ -33,10 +33,15 @@ var (
 	userAgent = fmt.Sprintf("SQL-Dataset/%s-%s", version, gitSHA)
 	maxRows   = 500
 
-	errUnexpectedResponse = errors.New("Unexpected server error response from Geckoboard")
-	errMoreRowsToSend     = "You're trying to send %d records, but we " +
-		"were only able to send the first %d. To send more, please " +
-		"change your dataset's update_type to 'append'"
+	errInvalidPayload = "There was an error sending the data to Geckoboard's API: %s"
+
+	errUnexpectedResponse = errors.New("Sorry, there seems to be a problem with " +
+		"Geckoboard's servers. Please try again, or check" +
+		"https://geckoboard.statuspage.io")
+
+	errMoreRowsToSend = "You're trying to send %d records, but 'replace' " +
+		"mode only supports sending %d. To send more, please change " +
```

Comment:
Is this accurate? i.e. does SQL-Dataset make multiple `append` requests if there are more total records than are allowed in a single request?

Reply:
Yes @bennewell thats correct we handle that automatically by batching requests of upto 500 records for append.

But for replace we will only ever send the first 500 rows and display that error.

https://github.com/geckoboard/sql-dataset/blob/master/client.go#L77-L111